home/jobs/OMERO-web/config.xml

```diff
@@ -32,22 +32,22 @@
   <concurrentBuild>false</concurrentBuild>
   <builders>
     <hudson.tasks.Shell>
-      <command>OMERO_DIST=$HOME/OMERO.server
+      <command>
+OMERO_DIST=$HOME/OMERO.server
 if [ -e $OMERO_DIST ]; then
-
-	if [ -e $WORKSPACE/omeropy-virtualenv ]; then
-		source $WORKSPACE/omeropy-virtualenv/bin/activate
+	if [ -e $HOME/omero-virtualenv ]; then
+		source $HOME/omero-virtualenv/bin/activate
         eval $(bash /opt/multi-config.sh ice3.5)
-    	$OMERO_DIST/bin/omero web stop
+        sudo systemctl stop omero-web.service
```

Comment:
No objections here, but this couples us tightly to C7. Any way to do this so that if we move to another platform it will still work?